pkg/commands/policy/flags.go

```diff
@@ -22,12 +22,20 @@ import (
 )
 
 type EnforceFlags struct {
+	DryRun      bool
 	ResultsFile string
 }
 
 func (e *EnforceFlags) Register(set *cli.FlagSet) {
 	f := set.NewSection("ENFORCE OPTIONS")
 
+	f.BoolVar(&cli.BoolVar{
+		Name:    "dry-run",
+		Default: false,
+		Target:  &e.DryRun,
+		Usage:   "Skips assigning reviewers and only reports any violations found.",
```

Comment:
@gjonathanhong might wanna generalize this message or add a todo for future use cases where we build another "action" from a policy violation.